monai/visualize/utils.py

```diff
@@ -47,15 +48,19 @@ def matshow3d(
     Create a 3D volume figure as a grid of images.
 
     Args:
-        volume: 3D volume to display. Higher dimensional arrays will be reshaped into (-1, H, W).
+        volume: 3D volume to display. data shape can be `BCHWD`, `CHWD` or `HWD`.
+            Higher dimensional arrays will be reshaped into (-1, H, W, [C]), `C` depends on `channel_dim` arg.
             A list of channel-first (C, H[, W, D]) arrays can also be passed in,
             in which case they will be displayed as a padded and stacked volume.
         fig: matplotlib figure to use. If None, a new figure will be created.
         title: title of the figure.
         figsize: size of the figure.
         frames_per_row: number of frames to display in each row. If None, sqrt(firstdim) will be used.
-        frame_dim: for higher dimensional arrays, which dimension (`-1`, `-2`, `-3`) is moved to the `-3`
+        frame_dim: for higher dimensional arrays, which dimension (`-1`, `-2`, `-3`) is moved to the `-3`.
```

Comment:
"moved to the `-3`." -> "moved to the `-3` dimension".

Reply:
updated in the latest commit.

Thanks.